src/components/questionnaire/questionnaires-edit-mode.tsx

```diff
@@ -237,8 +292,63 @@ const QuestionnairesEditMode = ({ questionnaire }: Props) => {
             value={editedQuestionnaire.description || ""}
             onChange={handleChange}
             fullWidth
-            sx={{ mb: 4 }}
+            sx={{ mb: 2 }}
           />
+          
+          {/* Tags Section */}
+          <Box sx={{ mb: 4 }}>
+            <Typography variant="subtitle1" sx={{ mb: 1 }}>{(strings as any).tags?.title || "Tags"}</Typography>
```

Comment:
Do we have to cast this as any, we should avoid using any where possible in typescript